rules/hmap.bzl

```diff
@@ -22,10 +22,13 @@ def _make_headermap_input_file(namespace, hdrs, flatten_headers):
     """
     entries = []
     for hdr in hdrs:
+        path = hdr.path
+        if path.endswith("-Swift.h"):
```

Comment:
Here again. We should not assume always generated header has suffix -`-Swift.h`.